cmd/mattermost/commands/config.go

```diff
@@ -68,15 +68,27 @@ var ConfigSetCmd = &cobra.Command{
 	RunE:    configSetCmdF,
 }
 
+var MigrateConfigCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Migrate config file",
+	Long:  "Instantiates a new config store and sets it based on an existing defined store",
+	Example: `	config migrate --from=path/to/config.json --to=a/different/path/to/configuration.json
+config migrate --to="postgres://mmuser:mostest@dockerhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"`,
```

Comment:
This is the better example, I think, but should have a sample `--from` shown explicitly.